alert-policies/nextcloud/MemoryUsage.yml

```diff
@@ -1,11 +1,11 @@
-name: Heap Memory Usage
+name: Memory Usage 
 
 description: |+
-  This alert is triggered  when the Heap Memory Usage is below 2 for at least 5 minutes.
+  This alert is triggered  when the Memory Usage is exceeds 90% for at least 5 minutes.
```

Comment:
```suggestion
  This alert is triggered  when the Memory Usage exceeds 90% for at least 5 minutes.
```

Reply:
Hi @mdumpati ,
As per your comments, I have made changes. Kindly review it.
Thanks...